eng/Packages.props

```diff
@@ -18,6 +18,7 @@
     <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.16.30" />
     <PackageReference Update="Microsoft.Win32.Registry" Version="4.3.0" />
     <PackageReference Update="NuGet.Build.Tasks" Version="$(NuGetBuildTasksVersion)" />
+    <PackageReference Update="NuGet.Frameworks" Version="$(NuGetBuildTasksVersion)" />
```

Comment:
Since this reference is used for dev code and not just test/bootstrap building, we'll need to add a darc subscription. This will be . . . unfun, because we'll have to juggle it across the many NuGet/VS/MSBuild releases between now and .NET 5 GA. However, we may be forced to avoid this by the VS Setup problems--if they're only resolvable via reflection-loading NuGet we don't have to have the direct reference anyway.